packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -503,7 +500,7 @@ class FlutterPlugin implements Plugin<Project> {
             // Warning: The name of this task is used by AndroidBuilder.outputsAppLinkSettings
             project.tasks.register("output${variant.name.capitalize()}AppLinkSettings") {
                 description "stores app links settings for the given build variant of this Android project into a json file."
-                variant.outputs.all { output ->
+                variant.outputs.configureEach { output ->
```

Comment:
android studio linter is prompting `Consider using 'configureEach' to avoid unnecessary configuration`

Reply:
@gmackall I feel like we have had to revert configureEach conversions before but cant recall the circumstances.